src/CatGFX.cpp

```diff
@@ -1,5 +1,18 @@
 #include "CatGFX.h"
 
+CatPrinter::CatPrinter(uint16_t h): 
+  Adafruit_GFX(384, h),
+  WIDTH_BYTE((WIDTH + 7)/8),
+  SERVICE_UUID("0000AE30-0000-1000-8000-00805F9B34FB"),
+  CHAR_UUID_DATA("0000AE01-0000-1000-8000-00805F9B34FB")
+{
+	  strcpy(this->printer_names[0], "GT01");
+	  strcpy(this->printer_names[1], "GB01");
+	  strcpy(this->printer_names[2], "GB02");
+	  strcpy(this->printer_names[3], "MX09");
+	  strcpy(this->printer_names[4], "");
```

Comment:
This leaves printer_names[5] uninitialized, so maybe just call resetNameArray first and then only add the 4 default printer names?